guides/Live-ImplementationGuide-BaRS/Home/Build/Testing-and-Environments/ReceiverOnboarding.page.md

```diff
@@ -0,0 +1,80 @@
+---
+topic: receiveronboarding
+---
+
+## {{page-title}}
+
+BaRS uses TLS-MA to communicate with Receiving endpoints. Receiving endpoints need a certificate under the NHS Root CA to facilitate TLS-MA.  The receiver needs to follow these steps for Integration and Production environments.
+
+To onboard as a receiver follow these steps:
```

Comment:
Is 'onboard' the right term? I think 'connect' or 'access' might be more appropriate